python/mspasspy/db/schema.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 import yaml
-import schema
+from schema import Schema, Or, Optional, SchemaError
```

Comment:
Just a personal preference, but I think `from schema import Schema` here is confusing. It enables such a class: `mspasspy.db.schema.Schema`, which appears to be a class defined by mspasspy but actually isn't.

Reply:
The reason I am doing this is that I don't wanna import the whole library into this program, but what you suggested seems more reasonable to use the previous version.

Reply:
I don't think there is a difference in terms of memory usage between the two style of import lines, so this is indeed just a preference thing.